routes/index.js

```diff
@@ -1,9 +1,11 @@
 const express = require("express");
 const router = express.Router();
 
+
 /* GET home page. */
 router.get("/", function (req, res, next) {
-	res.render("index", { title: "Express" });
+	res.render("satyam", { title: "Leenesh"});
```

Comment:
Is this mistakenly checked in?

Reply:
Yes sir i was telling the team members how to do a pull request and add reviewers so i pushed this one